demo-shell/src/app/components/form/form.component.html

```diff
@@ -3,46 +3,36 @@
     <mat-tab-group [animationDuration]="'0'">
         <mat-tab label="Form">
             <div class="app-form-container">
-                <adf-form
-                    [showRefreshButton]="false"
-                    [form]="form"
-                    (formError)="logErrors($event)">
+                <adf-form [showRefreshButton]="false" [form]="form" (formError)="logErrors($event)">
                 </adf-form>
             </div>
 
             <div class="app-console" #console>
                 <h3>Error log:</h3>
-                <p *ngFor="let error of errorFields">Error {{ error.name }} {{error.validationSummary.message | translate}}</p>
+                <p *ngFor="let error of errorFields">Error {{ error.name }} {{error.validationSummary.message |
+                    translate}}</p>
             </div>
         </mat-tab>
         <mat-tab label="Editor">
-            <ngx-monaco-editor
-                id="adf-form-config-editor"
-                class="app-form-config-editor"
-                [options]="editorOptions"
-                [(ngModel)]="formConfig"
-                (onInit)="onInitFormEditor($event)">
-            </ngx-monaco-editor>
+            <div class="app-form-config-editor">
+                <ngx-monaco-editor style="height: 100%" id="adf-form-config-editor" [options]="editorOptions"
```

Comment:
Can you add this style via a class, or using the id already mentioned here

```
#adf-form-config-editor {
    height: 100%;
}
````

Reply:
Done